src/resources/postcss/tickets/_block.pcss

```diff
@@ -100,8 +100,8 @@
 			}
 		}
 
-		&.tribe-tickets__item--price-sufix,
-		&.tribe-tickets__item--price-sufix.show-optout {
+		&.tribe-tickets__item--price-suffix,
+		&.tribe-tickets__item--price-suffix.show-optout {
```

Comment:
If we change this here, we break the old view which has the misspelling "sufix" in the class name there.

Reply:
No, actually we saved the v1 version as `tickets-v1.pcss` so we are free to do this and anything we want. Also, eventually change class names (Something I thought of, in order to make the structure more BEM friendly). 

https://github.com/moderntribe/event-tickets/blob/bucket/ari-js-normalization/src/resources/postcss/tickets-v1.pcss